packages/backend/discovery/kroma/ethereum/diffHistory.md

```diff
@@ -1,3 +1,25 @@
+Generated with discovered.json: 0x0bfa30e237d7f814492d4d1ab8191992240d15c4
+
+# Diff at Tue, 12 Mar 2024 10:43:25 GMT:
+
+- author: sekuba (<sekuba@users.noreply.github.com>)
+- comparing to: main@aa756f53b5067168dd1bce4c3f562d0212b0da15 block: 19412001
+- current block number: 19418455
+
+## Description
+
+Kroma mainnet fork occured at block #8171899. Team is announcing a rollback.
```

Comment:
it's not clear whether this challenge has anything to do with the fork. it probably does, we cannot say that without further investigation. just say that a root is challenged.